src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.parser;
+
+/**
+ * Contains Command Line Interface (CLI) syntax definitions common to multiple commands.
+ */
+public class RecipeBookSyntax {
+
+    /* Prefix definitions */
+    public static final Prefix PREFIX_NAME = new Prefix("-n");
+    public static final Prefix PREFIX_INGREDIENT = new Prefix("-i");
+    public static final Prefix PREFIX_STEPS = new Prefix("-s");
+    public static final Prefix PREFIX_TAG = new Prefix("--tag");
+    public static final Prefix PREFIX_DURATION = new Prefix("--time");
```

Comment:
Do we wna consider having a consistent format for all prefixes? Might be confusing/inconvenient to users to remember the different prefix formats e.g -t vs --tag

Reply:
agree since both tag and time starts w t, possibly use -t for tag and -d for duration?